easybuild/easyconfigs/o/OpenMPI/OpenMPI-2.0.2-GCC-6.3.0-2.27.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenMPI'
+version = '2.0.2'
+
+homepage = 'http://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-2 implementation."""
+
+toolchain = {'name': 'GCC', 'version': '6.3.0-2.27'}
+
+source_urls = ['http://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads']
+sources = [SOURCELOWER_TAR_GZ]
+sources = ['openmpi-%(version)src2.tar.gz']  # FIXME
```

Comment:
Release of OpenMPI 2.0.2 is further away than I thought it would be, see https://github.com/open-mpi/ompi/issues/2579#issuecomment-270751044 ...

I still feel we should wait it out though, especially since we still need to deal with FFTW and test the current `2017a` toolchain definitions thoroughly.